orbitx/flight_gui.py

```diff
@@ -14,9 +14,11 @@
 import math
 
 from . import common
-from . import orbitx_pb2 as protos
+from . import orbitx_pb2 as protos  # physics module
 
-import vpython
+import vpython                      # python 3D graphic library module
+import orbitx.style as style        # HTML5, Javascript and CSS3 code for UI
+import orbitx.Vpython as vp
```

Comment:
New modules should not have capitals. PEP8 (the python style guide that all python code uses) has a section on this: https://www.python.org/dev/peps/pep-0008/#package-and-module-names